dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/ScriptRunner.java

```diff
@@ -169,13 +169,13 @@ private void runScript(Connection conn, Reader reader) throws IOException, SQLEx
                             if (stopOnError && rs != null) {
                                 ResultSetMetaData md = rs.getMetaData();
                                 int cols = md.getColumnCount();
-                                for (int i = 0; i < cols; i++) {
+                                for (int i = 1; i < cols; i++) {
```

Comment:
why change this place,if change this,Whether execute success when upgrade in mysql ?

Reply:
because in pgsql: The number of columns starts from "1" and cannot start from "0". 
i have test this modify in mysql-5.6 and pg-9.6

Reply:
OK,It's greate!